react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/GutenbergBridgeJS2Parent.java

```diff
@@ -7,5 +7,14 @@
         void onMediaSelected(String mediaUrl);
     }
 
+    interface MediaUploadCallback {
+        void onUploadMediaFileSelected(String mediaId, String mediaUri);
+        void onMediaFileUploadProgress(String mediaId, float progress);
+        void onMediaFileUploadSucceeded(String mediaId, String mediaUrl);
+        void onMediaFileUploadFailed(String mediaId);
+    }
+
     void onMediaLibraryPress(MediaSelectedCallback mediaSelectedCallback);
+
+    void onUploadMediaPress(MediaUploadCallback mediaUploadCallback);
```

Comment:
Maybe we should follow the naming convention used in https://github.com/wordpress-mobile/WordPress-Android/pull/8879/files#diff-38b6aea9cfff5f3e0e4b7389b44de602R3185  and have the verb conjugation be in the past tense, i.e. `onUploadMediaButtonClicked` and `onMediaLibraryButtonClicked`, following the same pattern of `on<EventHappened>` as used in `interface EditorFragmentListener`

Reply:
Thanks @mzorz . Fixed!